python/lsst/dax/imgserv/getimage/skymapImage.py

```diff
@@ -0,0 +1,237 @@
+#
+# LSST Data Management System
+# Copyright 2017 AURA/LSST.
```

Comment:
I think 2015-2017 is more accurate as significant parts of the code are copied from skymapStich.py. Also, it would be desirable to delete skymapStitch.py, if reasonably possible. Having that much nearly duplicate code sitting around is likely to cause confusion.

Reply:
Ok, moving imageREST_v0 to use new SkymapImage, where all the logic has been preserved, and remove the old files.